### PR TITLE
Fix locale key for Wechatwork webhook

### PR DESCRIPTION
**Pull request by qwerty287**
The locale key in the locale files used for the Wechatwork webhook was different from the key used in the template ([here](https://github.com/go-gitea/gitea/blob/078e2b2c39f119b9cbe5678ce12f1cc9eb6ddda9/templates/repo/settings/webhook/wechatwork.tmpl#L2)). I changed it to fix this text.

### CODE REVIEW

**Comment by zeripath** [comment]
Why not just change the template instead?

File: options/locale/locale_zh-TW.ini [hunk 1]
```diff
@@ -1855,7 +1855,7 @@ settings.add_telegram_hook_desc=將 <a href="%s">Telegram</a> 整合到您的儲
 settings.add_matrix_hook_desc=將 <a href="%s">Matrix</a> 整合到您的儲存庫。
 settings.add_msteams_hook_desc=將 <a href="%s">Microsoft Teams</a> 整合到您的儲存庫。
 settings.add_feishu_hook_desc=將 <a href="%s">Feishu</a> 整合到您的儲存庫。
-settings.add_Wechat_hook_desc=將 <a href="%s">Wechatwork</a> 整合到您的儲存庫。
+settings.add_wechatwork_hook_desc=將 <a href="%s">Wechatwork</a> 整合到您的儲存庫。
```
Comment (techknowlogick) [inline options/locale/locale_zh-TW.ini]: non-english translations will be overwritten by the CI. Per @zeripath's suggestion, changing the template may be a better approach.

**Comment by qwerty287** [comment]
Would be possible too. I'll change it, wait a moment...

File: options/locale/locale_zh-TW.ini (hunk 1 above)
Comment (qwerty287) [reply options/locale/locale_zh-TW.ini]: Okay, fixed it.

**Review by zeripath — APPROVED** [review]

**Review by axifive — APPROVED** [review]

**Pull request merged**